lib/v2/threads/index.js

```diff
@@ -0,0 +1,219 @@
+            description += `<img src="${item.post.user?.profile_pic_url}" width="48px" height="48px"> `;
+        }
+        description += `<strong>@${item.post.user?.username}</strong>`;
+        if (embededPost) {
+            description += options.showEmojiForQuotesAndReply ? ' 🔁' : ' quoted';
+        } else if (isReply) {
+            description += options.showEmojiForQuotesAndReply ? ' ↩️' : ' replied';
+        }
+        description += ':</p>';
+    }
+
+    if (item.post.caption?.text) {
+        title += item.post.caption?.text;
+        description += `<p>${item.post.caption?.text}</p>`;
+    }
+
+    if (hasMedia(item.post)) {
+        description += `<p>${buildMedia(item.post)}</p>`;
+    }
+
+    if (embededPost) {
+        if (options.showQuotedInTitle) {
+            title += options.showEmojiForQuotesAndReply ? ' 🔁 ' : ' QT: ';
+            title += `@${embededPost.user?.username}: `;
+            title += `"${embededPost.caption?.text}"`;
+        }
+        description += '<blockquote>';
+        description += `<p>${embededPost.caption?.text}</p>`;
+        if (hasMedia(embededPost)) {
+            description += `<p>${buildMedia(embededPost)}</p>`;
+        }
+        description += '— ';
+        if (options.showQuotedAuthorAvatarInDesc) {
+            description += `<img src="${embededPost.user?.profile_pic_url}" width="24px" height="24px"> `;
+        }
+        description += `@${embededPost.user?.username} — `;
+        description += `<a href="${threadUrl(embededPost.code)}">${dayjs(embededPost.taken_at, 'X').toString()}</a>`;
+        description += '</blockquote>';
+    }
+    return { title, description };
+};
+
+module.exports = async (ctx) => {
+    const { user, routeParams } = ctx.params;
+    const { appId, lsd, userId } = await extractTokens(user, ctx);
+
+    const params = new URLSearchParams(routeParams);
+    ctx.state.json.params = routeParams;
+
+    const options = {
+        showAuthorInTitle: params.get('showAuthorInTitle') ?? true,
+        showAuthorInDesc: params.get('showAuthorInDesc') ?? true,
+        showAuthorAvatarInDesc: params.get('showAuthorAvatarInDesc') ?? true,
+        showQuotedInTitle: params.get('showQuotedInTitle') ?? true,
+        showQuotedAuthorAvatarInDesc: params.get('showQuotedAuthorAvatarInDesc') ?? true,
+        showEmojiForQuotesAndReply: params.get('showEmojiForQuotesAndReply') ?? true,
+        replies: params.get('replies') ?? false,
+    };
+
+    const headers = makeHeader(user, appId, lsd);
+    const body = makeBody({
+        lsd,
+        variables: encodeURIComponent(JSON.stringify({ userID: userId })),
+        doc_id: options.replies ? REPLIES_QUERY : THREADS_QUERY,
+    });
+
+    const resp = await got.post(apiUrl, {
+        headers,
+        body,
+    });
+
+    ctx.state.json.request = {
+        headers: resp.request.options.headers,
+        body: resp.request.options.body,
+    };
+
+    const responseBody = JSON.parse(resp.body);
```

Comment:
```suggestion
    const responseBody = resp.data;
```
`@/utils/got` will try to perform a JSON.parse and put it under .data
https://github.com/DIYgod/RSSHub/blob/b6e1cf22aca3aff02c9244e0f64a276373dd5aa2/lib/utils/got.js#L15-L19